apps/mobile/src/constants.ts

```diff
@@ -1,3 +1,2 @@
 export const TWITTER_LINK = 'https://twitter.com/LeatherBTC';
 export const BROWSER_EXTENSION_LINK = 'https://leather.io/install-extension';
-export const __NEW_SOME_KIND_OF_CONST = 'WHAT CONST?';
```

Comment:
I noticed this when re-checking https://github.com/leather-wallet/mono/pull/99 to see if I was doing the right thing